package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "binaris",
-  "version": "7.0.1",
+  "version": "8.0.0,
```

Gaaah!  No close quote.  (Breaks syntax highlighting in GitHub, but also probably fails compilation horribly...).
```suggestion
  "version": "8.0.0",
```